.gitignore

```diff
@@ -1 +1,2 @@
-.idea/
\ No newline at end of file
+.idea/
+node_modules/
```

Comment:
Maybe we should add package-lock here?

Reply:
`Package-lock.json` holds info about all sub-dependencies. Package.json only has direct dependencies. Usually, dependent packages themselves don't use exact versions for their dependencies and thus these sub-dependencies are not locked. In contrast, package-lock.json is kinda 'safe protection layer'.

I think we could leave it as is for now.